app/controllers/events/view/edit/attendee.js

```diff
@@ -0,0 +1,35 @@
+import Controller from '@ember/controller';
+import EventWizardMixin from 'open-event-frontend/mixins/event-wizard';
+
+export default Controller.extend(EventWizardMixin, {
+  async saveForms(data) {
+    for (const customForm of data.customForms ? data.customForms.toArray() : []) {
+      await customForm.save();
+    }
+    return data;
+  },
+  actions: {
+    async save(data) {
+      try {
+        await this.saveForms(data);
+        this.saveEventDataAndRedirectTo(
+          'events.view.index',
+          []
+        );
+      } catch (error) {
+        this.notify.error(this.ln10.t(error.message));
+      }
+    },
+    async move(direction, data) {
+      try {
+        await this.saveForms(data);
+        this.saveEventDataAndRedirectTo(
+          direction === 'forwards' ? 'events.view.edit.sponsors' : 'events.view.edit.basic-details',
+          []
+        );
+      } catch (error) {
+        this.notify.error(this.ln10.t(error.message));
```

Comment:
I guess this should be   `this.notify.error(this.l10n.t(error.message));`.

Reply:
@mrsaicharan1 Thanks for pointing out the typo, I have fixed it now. Please review